lib/iris/tests/unit/fileformats/grib/load_convert/test_scanning_mode.py

```diff
@@ -27,7 +27,7 @@
 import iris.tests as tests
 
 from iris.exceptions import TranslationError
-from iris.fileformats.grib._load_convert import scanning_mode, ScanningMode
+from iris.fileformats.grib._load_convert import ScanningMode, scanning_mode
```

Comment:
I suspect that it just got tacked onto the end, and their focus was not also ordering the symbol that's imported, but ordering the modules